docs/aggregation-layer/.pages

```diff
@@ -1,3 +1,4 @@
 nav:
   - 3.1. Version 0.2.0: v0.2.0
-  - 3.2. Version 0.3.0: v0.3.0
\ No newline at end of file
+  - 3.2. Version 0.3.0: v0.3.0
```

Comment:
I'd create a new section exclusively for SC.
- `4. Smart Contracts`
  - `4.1 v12`
  - `4.2 v13`
  - ...

Similar to this: https://github.com/agglayer/protocol-team-docs/pull/10/files#diff-5b7137a1b17cd50c3db509d7fccc84b453921a9ab979daa5e1a372fda4575a0dR6

Comment [here](https://github.com/agglayer/protocol-team-docs/pull/10/files#r2292999145)